css/home.css

```diff
@@ -1,18 +1,71 @@
 /* Home page specific styles */
+html,
+body {
+  margin: 0;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+.footer {
+  background-color: #003c8f;
+  color: white;
+  width: 100%;
+  padding: 40px;
+  margin: 0 auto;
+  overflow: hidden;
+  position: relative;
+  font-family: Roboto;
+  text-align: center;
+  font-size: 16px;
+  line-height: 21px;
+}
+
+.footer__footnote {
+  color: white;
+  position: relative;
+  display: inline-block;
+  text-decoration: none;
+}
+
+.footer__footnote::after {
+  content: "";
+  width: 100%;
+  height: 1px;
+  background-color: white;
+  display: block;
+  position: absolute;
+  left: 0;
+  bottom: -5px;
+  right: 0;
+  margin: auto;
+}
+
+.footer__icons-container {
+  margin: 30px 5px;
+}
+.footer__icon {
+  margin: 7px;
+}
 
 .homepage-hero {
+  position: absolute;
   top: 0;
   right: 0;
   bottom: 0;
   left: 0;
   width: 100%;
-  height: 100vh;
+  height: 100%;
   font-family: "roboto", sans-serif;
   background: url("../assets/images/frente1.JPG") no-repeat center;
   background-size: cover;
 }
 
 .homepage-hero__layer {
+  width: 100%;
```

Comment:
why are you adding a `width` here?

Reply:
i no have any idea, i just copy and paste the code from develop nothing more.

Reply:
please remove it, as gh thinks it is yours

Reply:
done.